pickATrip_django_apps/templates/pickATrip_django_apps/base.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+	<!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+	integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+	<title>{{title}}</title>
+</head>
+<body>
+
```

Comment:
I think a sidebar might a GOOD IDEA to add to the base.html file- like this example: 
![sidebar](https://user-images.githubusercontent.com/48455448/113256820-6201ba80-92d2-11eb-8fa8-5164847159b8.PNG)


only I recommend if we do so, just use the a tag with a more generic way of referencing the path (URL). by using this capability of django - <a href="{% url 'urlname' %}">, when URL name is the name we gave to the URL in the URL file of our project and maybe we need to think what portion of the page we would like it to be

Reply:
Yes we will add a sidebar, I thought we should add it in a different PR and also adding some bootstrap features that will make our application looks better.